gettsim/parameters/eink_st.yaml

```diff
@@ -239,3 +239,12 @@ ertragsanteil:
       upper_threshold: inf
       rate_linear: 0.0
       intercept_at_lower_threshold: 1
+
+rounding:
```

Comment:
Oh, so you did add some rounding? Sorry I missed that in the previous round. Be precise then in changes.rst